src/routes/+page.ts

```diff
@@ -27,25 +33,50 @@ async function requestProjectMetadata(project: Project): Promise<ProjectMetadata
     };
 }
 
+async function requestUndecidedPapers(project: Project): Promise<PaperInfos[]> {
+    const projectController = BackendController.getInstance().project(project.id);
+    const currentStage: number = await projectController.getCurrentStage();
+    const latestStage: number = await projectController.getStageCount();
+    const allUndecidedPapers: PaperInfos[] = [];
+
+    for (let i = currentStage; i <= latestStage; i++) {
+        const allStageEntriesFromStageI: StageEntry[] = await projectController
+            .stage(currentStage)
```

Comment:
That should be `i` instead of `currentStage`
```suggestion
            .stage(i)
```

Reply:
You are right, thank you!